src/pivot-table/components/cells/TotalsCell.tsx

```diff
@@ -25,6 +25,7 @@ const TotalsCell = ({ cell, style, isLeftColumn, isLastRow, isLastColumn }: Labe
 
   return (
     <div
+      title={cell.ref.qText}
```

Comment:
I think getting qText from ref can be resulted in undefined right?

Reply:
Yes then the attribute should not be rendered by react